rpc/rpc.go

```diff
@@ -6,16 +6,15 @@ import (
 	"strings"
 	"sync"
 
-	"go.uber.org/zap"
-
 	"github.com/qlcchain/go-qlc/config"
 	"github.com/qlcchain/go-qlc/consensus"
 	"github.com/qlcchain/go-qlc/ledger"
 	"github.com/qlcchain/go-qlc/log"
 	"github.com/qlcchain/go-qlc/wallet"
+	"go.uber.org/zap"
 )
 
-//var logger = log.NewLogger("rpc")
+var logger = log.NewLogger("rpc")
```

Comment:
you should not create a global logger,  it will init before LogServie init, so the log config does not work.

Reply:
i have added a `logger` field to `jsonCodec`